Portal/Views/_ViewImports.cshtml

```diff
@@ -1,4 +1,5 @@
 ﻿@using PRN.Web
 @using PRN.Web.Models
+@using PRN.Web.Constants
```

Comment:
This line is not needed

Reply:
This is needed, otherwise we get compilation errors.